config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml

```diff
@@ -41,8 +41,7 @@ periodics:
       args:
       - bash
       - -c
-      - >
-        ./test/build.sh install-lib &&
+      - ./test/build.sh install-lib &&
```

Comment:
hummm are you sure this will work? (everything in args will be plumbing through a go exec.Command)

Reply:
I think so. It is almost the same as using `>`. The difference is that `>` preserves trailing spaces, and we don't really have trailing spaces here. See https://yaml-multiline.info/ and https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines.

With that being said, I think it is probably worthwhile to keep `>` here for better readability. So I changed it back.

Reply:
I decided to use `>-`, which is block style for better readability and also removes the final newline.